firstvoices/backend/views/join_request_views.py

```diff
@@ -145,6 +147,25 @@ def get_queryset(self):
             "site", "site__language", "created_by", "last_modified_by", "user"
         )
 
+    def get_validated_site(self):
```

Comment:
The join request model already has a create permission. I worry that adding join permission on the site will get confusing. Maybe a simpler option here would be to skip the new permission, and just adjust get_validated_site to skip the site permission check for the "create" action